src/autorest.bicep/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "skipLibCheck": true,
+    "alwaysStrict": true,
+    "forceConsistentCasingInFileNames": false,
+    "module": "commonjs",
+    "noEmitOnError": true,
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "sourceMap": true,
+    "declarationMap": true,
+    "strict": true,
```

Comment:
Btw `"strict": true` implies `"alwaysStrict": true`, `"noImplictAny": true` and `"noImplictThis": true` so they could safely be removed (it doesn't hurt to keep them though).